### ISSUE TITLE
Ignore missing JSDoc parameter type

### ISSUE THREAD

**Issue by armorik83** [eslint/eslint#2457]
Hi.

I want to way to ignore "Missing JSDoc parameter type for Foo" because I have referred to the type from other sources in using WebStorm.

https://github.com/eslint/eslint/blob/master/lib/rules/valid-jsdoc.js#L97-L98

Would you add something new options for avoid the warning?

**Comment by gyandeeps** [comment]
Why do you want to do that? Is't that be a anti-pattern?
To achieve this just deactivate the rule around that piece.

**Comment by nzakas** [comment]
@armorik83 we need a better description of what you're asking for

**Comment by armorik83** [comment]
I refer to another file for type using WebStorm just like in the TypeScript. At this time, although not want strict verification of the type, I wanted received only the `prefer` option benefits.
Certainly it might be complexity, thanks.

**Comment by nzakas** [comment]
Oh I see, you're asking for an option where we don't warn if the parameter type isn't specified. In assuming you would also want this to apply for `@returns` as well?

**Comment by armorik83** [comment]
Yes master. I wish to encourage to follow the `@returns`, not that I want to get the warn about the parameter type so. I also understand that it is an illegal type strictly.

**Comment by nzakas** [comment]
So `@returns` should still have a type? You only want to ignore the type for `@param`?

**Comment by armorik83** [comment]
Well, It does not work even a type that WebStorm recognizes. I am glad when the type strictness in everywhere is relaxed.

I was optimistic, write more detail. WebStorm is not well interpret the `@typedef` unfortunately. When trivial, I make the lazy have not write types. This does not meet the specifications. However it is also not the type any.

```js
// The function for CLI

/**
 * @param opts  // mm.. is this any?
 * @param args
 * @return {Promise} // validates the here @return's'
 */
```

But yet, should I write `@param {*} opts`, `@param {*} args`?

**Comment by gyandeeps** [comment]
just to get the convo gng here, @armorik83 you are specifically talking about webstorm usage (i also use webstorm i find it helpfull tho) but what about all the jsdoc generators who make an html site for your docs. they use it.

**Comment by gyandeeps** [comment]
Feel free to reopen for further discussions.
